osgar/drivers/lora.py

```diff
@@ -37,11 +37,14 @@ def parse_lora_packet(packet):
     """
     s = packet.split(b'|')
     try:
-        return [int(x) for x in s[:-1]], s[-1].strip()
+        addr, data = [int(x) for x in s[:-1]], s[-1].strip()
     except ValueError:
-        print('Invalid LoRa packet:', packet)
         return None, packet
 
+    if len(addr) < 1 or addr[0] not in range(1, 6):       
```

Comment:
It is not probably significant but this creates an iterator that iterates through the six values and comparing them with addr[0]. Just testing addr[0] >= 1 and addr[0] < 6 would be faster. It might be more significant in other cases where the continuous range of values is larger.